.github/workflows/test.yml

```diff
@@ -0,0 +1,28 @@
+name: python-bsonjs
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: ["2.7", "3.4", "3.5", "3.6", "3.7", "3.8", "3.9", "3.10"]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      # have to manually install pymongo for 3.4 and 3.5 because otherwise it does not work
```

Comment:
What is the error message? It may be helpful to add it to the comment.

Reply:
Done.

Reply:
Uh that's strange but we're about to drop support for <3.6 so there's need to look into it further.